bin/resty

```diff
@@ -475,6 +482,8 @@ Options:
     --main-include path Include the specified file in the nginx main configuration block
                         (multiple instances are supported).
 
+    --ns IP             Specify the nameserver(s) (multiple instances are supported)
```

Comment:
My original regex supported multiple with one flag, but I will change this. I debated name server vs nameserver and the former feels generic, but the later is the thing (a technical entity), but whatever I can change them back.